modules/nw-bgp-routing-config.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp-routing/configuring-bgp-routing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-bgp-routing-config_{context}"]
+= Configuring BGP routing
+
+As a cluster administrator, you can enable or disable BGP routing support for your cluster on bare-metal infrastructure.
+
+[id="enabling-bgp-routing-support_{context}"]
+== Enabling BGP routing support
+
+As a cluster administrator, you can enable Border Gateway Protocol (BGP) routing support for your cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Cluster is installed on bare-metal infrastructure.
+
+.Procedure
+
+. To enable dynamic routing, enter the following command:
++
+[source,terminal]
+----
+$ oc patch network.operator cluster -p '{
+  "spec": {
+    "defaultNetwork": {
+      "ovnKubernetesConfig": {
+        "routeAdvertisements": "Enabled"
+      }
+    }
+  }
+}'
+----
+
+. To enable a dynamic routing provider, enter the following command:
```

Comment:
This would be done first, so I would probably move it ahead of routeAdvertisements flag above